res/locales/it/stopwords.txt

```diff
@@ -0,0 +1,98 @@
+a

```

Comment:
You need to put an Italian list of stopwords, you can search on this repository:
https://github.com/stopwords-iso/stopwords-it